src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/controllers/web/WorkflowRunController.kt

```diff
@@ -49,6 +49,35 @@ class WorkflowRunController(
         )
     }
 
+    fun getWorkflowRunSummary(): String
+    {
+        // val key = context.params(":key")
+        val workflowRunSummaryRequestJson = context.getRequestBody()
+
+        val workflowRunSummaryRequest = try
+        {
+            Serializer.instance.gson.fromJson(workflowRunSummaryRequestJson, WorkflowRunSummaryPageRequest::class.java)
+        }
+        catch (e: JsonSyntaxException)
+        {
+            throw BadRequest("Invalid workflow description: ${e.message}")
+        }
+        // val requestBody = context.getRequestBody()
+        // val details = workflowRunRepository.getWorkflowRunDetails(key)
+        val reports: MutableList<WorkflowRunSummaryPageReport> = mutableListOf()
+        for (report in workflowRunSummaryRequest.reports) {
+            reports.add(WorkflowRunSummaryPageReport(report.name, null, report.params, null))
+        }
+        // val ref = details.gitCommit
+        val body = Serializer.instance.gson.toJson(WorkflowRunSummaryPageRequest(reports, null))
+        val response = orderlyServerAPI.post(
+            "/v1/workflow/summary/",
+            body,
+            emptyMap()
+        )
```

Comment:
```suggestion
        val response = orderlyServerAPI.post(
    "/v1/workflow/summary/",
    context.getRequestBody(),
    emptyMap()
)
```
This can be much simpler, you literally should just be able to pass through the request body as is - this worked for me.

Also, orderly web returns 500s if the report or ref isn't known, so you need to use a report which exists, and in the integration test use `getGitBranchCommit` to get a real ref for 'other' or 'master/